packages/notebook/test/actions.spec.ts

```diff
@@ -1698,6 +1698,117 @@ describe('@jupyterlab/notebook', () => {
         NotebookActions.undo(widget);
         expect(widget.widgets.length).toBe(count - 2);
       });
+
+      it('should emit a signal with cut action', async () => {
+        let signals: {
+          action: 'copy' | 'cut' | 'paste' | null;
+          count: number;
+        }[] = [];
```

Comment:
If that was an interface, the test would have fewer repetitive bits like this too.